src/tests/ffi.cpp

```diff
@@ -6560,18 +6566,20 @@ TEST_F(rnp_tests, test_ffi_op_verify_sig_count)
     rnp_input_destroy(input);
 
     /* malformed detached signature, wrong bitlen in MPI  */
+    sigcount = 255;
     assert_rnp_success(rnp_input_from_path(&source, "data/test_messages/message.txt"));
     assert_rnp_success(
       rnp_input_from_path(&input, "data/test_messages/message.txt.sig.wrong-mpi-bitlen"));
     assert_rnp_success(rnp_op_verify_detached_create(&verify, ffi, source, input));
-    assert_int_equal(rnp_op_verify_execute(verify), RNP_ERROR_BAD_PARAMETERS);
+    assert_int_equal(rnp_op_verify_execute(verify), RNP_SUCCESS);
```

Comment:
we usually use `assert_rnp_success`

Reply:
@rrrooommmaaa Agree, just copy-pasted piece of some old code, written before assert_rnp_success was introduced. Will fix and force-push.